### ISSUE TITLE
Digital Signature has no company name

### ISSUE THREAD

**Issue by hoggatt** [HeidiSQL/HeidiSQL#1681]
**Preconditions**

* HeidiSQL version: HeidiSQL 12.1.0.6537 64 Bit
* Database type and version: N/A
* OS: Windows 10

**Describe the bug**

The digital signature of heidisql.exe has an unspecified company:
```
Verified:	Signed
Signing date:	1:06 AM 8/13/2022
Publisher:	Ansgar Becker
Company:	n/a
Description:	HeidiSQL 12.1.0.6537 64 Bit
Product:	HeidiSQL
Prod version:	n/a
File version:	n/a
MachineType:	64-bit
Binary Version:	12.1.0.6537
Original Name:	n/a
Internal Name:	n/a
Copyright:	See GPL license
Comments:	n/a
Entropy:	6.363
MD5:	BE5D753187C3F7D2C8DD9AAB78F5EA3C
SHA1:	A37855A1C43439A7090654A7BC84C60EEECB8005
PESHA1:	41E91BABE85E0CF0939592A37AEFD906FB3BB5A0
PE256:	E1D1645827FFAB4FDE0FCD30CA569B84FA8A1D1E97EFD04FD9235A9F4E26DEB1
SHA256:	D56F3EAD1EB2D28230C40D075F31AA0EBF642CC9429C57D36AADAFDF0EB8AF57
IMP:	AEAF1D5777EEEE484FDE037248CE8283
```

This is an issue at our company because we want to whitelist heidisql.exe for our engineers, but since your digital signature doesn't have a company name, we can only whitelist specific versions using the MD5 hash. 

Would it be possible to put something in the 'Company' field so that we can whitelist every version of your awesome software? 

**To Reproduce**

N/A

**Screenshots or Crash reports**

N/A

**Comment by ansgarbecker** [comment]
The publisher name should be sufficient for whitelisting, or?

I highly guess there is no way of modifying an existing certificate, not least for security reasons. These certificates own a creation timestamp, which is double-checked each time when it's used for signing.
For the next certificate I have to buy, I can of course verify there's a company name in it. But my current certificate runs out in two years if I recall right.

**Comment by hoggatt** [comment]
I agree with you that it doesn't make sense, but the "security" software they use only checks the company field. 

Thanks for being willing to put that in there next time. If you remember to ping this issue when the change is made, that would be awesome. Otherwise I'll try to check back in mid-2024!

**Comment by ansgarbecker** [comment]
I thought I can drop Sectigo, the certificate provider, a question why my company name is not included in the certificate. The company field in my details is filled as "Ansgar Becker Software Development".
But I'm confused - isn't the "O" part meant for "Organization"? Or where should the company name be placed? The "O" part in the certificate already has my firstname/lastname, but it should contain the company name:
![grafik](https://user-images.githubusercontent.com/7986591/195972684-d196c1d2-42d5-4f8f-9c9a-837d6dd0d450.png)

**Comment by hoggatt** [comment]
That's a great question. That appears to be the case, but if you use [Microsoft's digital signature checker](https://learn.microsoft.com/en-us/sysinternals/downloads/sigcheck) like so:
`sigcheck64.exe -a -e -h -s . > SigCheckLog.txt`

You'll get the output I have in my first post showing no company name. 

I've attached a .zip that has the heidi exe, the sigcheck64 program, and a .bat script to generate the .txt file:
[Heidi_Sig_Check.zip](https://github.com/HeidiSQL/HeidiSQL/files/9814042/Heidi_Sig_Check.zip)

**Comment by ansgarbecker** [comment]
The certificate has changed, I payed 700 EUR for a new one last year. But the company field is still empty with the latest signed build:

```
heidisql64.exe:
	Verified:	Signed
	Signing date:	11:04 09.02.2026
	Publisher:	Ansgar Becker
	Company:	n/a
	Description:	HeidiSQL 12.15.0.7179 64 Bit
	Product:	HeidiSQL
	Prod version:	n/a
	File version:	n/a
	MachineType:	64-bit
	Binary Version:	12.15.0.7179
```

@hoggat could you please check if that's still a problem on your side? I would otherwise like to close the issue.

**Comment by hoggatt** [comment]
Thank you for trying! I will close since this is no longer an issue for us.
